.golangci.yml

```diff
@@ -95,8 +95,16 @@ linters-settings:
       - kilometre
       - kilometres
 
+  depguard:
+    list-type: denylist
+    include-go-root: true
+    packages-with-error-message:
+      # See https://github.com/open-telemetry/opentelemetry-collector/issues/5200 for rationale
+      - sync/atomic: "Use go.uber.org/atomic instead of sync/atomic"
```

Comment:
nice! I've not used `depguard` before

Reply:
Me neither, thanks to @hanjm for proposing it here https://github.com/open-telemetry/opentelemetry-collector/pull/5181#discussion_r848974947 :)